device_examples/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+function(add_example example_name)
+  add_executable(${example_name} ${example_name}.cpp)
+  target_link_libraries(${example_name} spblas fmt)
+  target_compile_options(${example_name} PRIVATE "-Wno-unused-result")
+endfunction()
+
+add_example(simple_spmv)
```

Comment:
I guess these example files need to have `cuda` in the name or be in a CUDA directory? They would clash with other backends (like AMD).

Reply:
Also, it's weird that you don't at least link to `cudart` despite using explicit calls to `cuda` in the example? You rely on the `spblas` target to give this to you currently.